chainermn/datasets/empty_dataset.py

```diff
@@ -0,0 +1,21 @@
+import chainer
+
+
+def get_empty_dataset(dataset):
```

Comment:
How about changing its name to `create_empty_dataset`, which is consistent to `create_multi_node_optimizer` and `create_multi_node_evaluator`?